api/server.spec.js

```diff
@@ -18,14 +19,16 @@ describe('/ [GET]', () => {
         expect(res.body.message).toEqual(
           'The Where-to-Code Server is up!',
         );
+        done()
       }));
 
-  it('catches and handles bad request', () =>
+  it('catches and handles bad request', (done) =>
     request(server)
       .get('/very-inappropriate-url')
       .expect('Content-Type', /json/)
       .expect(404)
       .then(res => {
         expect(res.body.message).toEqual('No endpoint matches that URL.');
+        done()
```

Comment:
Add a semi-colon.
Adhere to eslint rules.

Reply:
Done. Semicolon and configured eslint